homepage.html

```diff
@@ -51,6 +52,44 @@ <h2 id="supportingtext">I'm a Software Developer</h2>
                 <img src="images/illustration.png" alt="Illustration">
             </div>
         </section>
+        <section class="recentworks">
+            <h1 class="recent">My Recent Works</h1>
+            <div class="vector">
+                <img src="images/Vector.png" alt="line">
+            </div>
+            <div class="gymfit">
+                <img src="images/gymfit.png" alt="gymfit">
+            </div>
+            <h2 class="stories">Multi-Post Stories</h2>
+            <p class="passage">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="tags">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>JavaScript</li>
+            </ul>
+            <button id="show" class="button">See Project</button>
+            <div class="bg">
+                <img src="images/bg.png" alt="bg">
+            </div>
+
+            <h2 class="stories2">Professional Art Printing Data</h2>
+            <p class="passage2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="tags2">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>JavaScript</li>
+            </ul>
+            <button id="show2" class="button">See Project</button>
+
+        </section>
+        <section class="about">
+            <h1 class="aboutme">About me</h1>
+            <p class="passage3">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button id="resume" class="button">Get My Resume</button>
+            <div class="frame">
+                <img src="images/Frame.png" alt="frame">
+            </div>
+        </section>
```

Comment:
- [ ] it is a requirement the image should be set as a background image with CSS
- [ ] Also have the contents of this section aligned with flexbox
- [ ] You happen to be missing the technologies. Kindly add them since this is a requirement.